### PR TITLE
[Tadhg] Adding the rest of the servies to the pipeline

### PR DESCRIPTION
**Pull request by tadhgryan2012**
## Summary
Added the rest of the services to the pipeline

## How to verify
If pipeline passes

## Related

### CODE REVIEW

**Comment by tadhgryan2012** [comment]
This will pass when the eureka gets accepted

**Review by s1m0nWasTaken — APPROVED** [review]
not good enough - doesnt work !!!!!!

**Review by oOoOoOo0o — APPROVED** [review]
Looks good but needs to wait for EurekaService to be merged before merging

**Pull request merged**
